x-pack/plugin/snapshot-based-recoveries/src/internalClusterTest/java/org/elasticsearch/xpack/snapshotbasedrecoveries/recovery/SnapshotBasedIndexRecoveryIT.java

```diff
@@ -740,7 +735,7 @@ public void sendResponse(Exception exception) throws IOException {
 
                             private boolean assertShardClosedException(Exception e) {
                                 if (e instanceof CancellableThreads.ExecutionCancelledException cancelledException) {
-                                    assert cancelledException.getMessage().contains("shard closed") : e;
+                                    assert cancelledException.getMessage().contains("shard is closed") : e;
```

Comment:
Original change that introduced this assertion: https://github.com/elastic/elasticsearch/pull/97096
Please note, the error mentioned there contains "shard is closed". 
This makes me believe that "is" was lost accidentaly.